src/components/Profile/index.js

```diff
@@ -22,6 +22,14 @@ const Profile = props => {
     history,
   } = props
 
+  const [returnToTutorial, setReturnToTutorial] = useState(false)
+
+  useEffect(() => {
+    if (history.location.state && history.location.state.navigateBack) {
```

Comment:
Missing dependency in useEffect

Reply:
Excellent point! That would definitely have been a great idea, but I don't think there's time to do that now.. I will fix the dependency.